frameworks/person-escort-record/questions/property-bags.yml

```diff
@@ -0,0 +1,11 @@
+type: add_multiple_items
+question: Bags of property
+list_item_name: Bag
+questions:
+  - property-bag-seal-number
+  - property-bag-type
+validations:
+  -
+    type: minlength
```

Comment:
@cwrw so it turns out I'm really struggling with this. It appears the [`minlength` validator](https://github.com/UKHomeOffice/passports-form-wizard/blob/9dd5356e7d62c3c4050ed83e43a8a2c543577429/lib/validation/validators.js#L29) actually expects a string and validates the minimum number of characters.

I tried to see if I could patch this validator but I then discovered they appear to [send in the options from an array individually](https://github.com/UKHomeOffice/passports-form-wizard/blob/9dd5356e7d62c3c4050ed83e43a8a2c543577429/lib/validation/index.js#L100-L103) when validating fields.

This causes problems when trying to validate the length of the array as the validator doesn't seem to have access to it. I've spent quite a bit of time on this and didn't want to waste much more so looked for alternatives.

Would switching this validator to `required` still work for you? I was able to get it working as expected with that validator.